client/app/pages/queries/add-to-dashboard.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal-header">
+    <button type="button" class="close" aria-label="Close" ng-click="$ctrl.close()"><span aria-hidden="true">&times;</span></button>
+    <h4 class="modal-title">Add to Dashboard</h4>
+</div>
+<div class="modal-body">
+    <form name="alertForm" class="form">
+      <div class="form-group">
+        <!-- <label>Limit Dashboard search to dashboards I've created? </label>
+        <input type="checkbox" ng-model="$ctrl.limitToUsersDashboards" on-change="$ctrl.dashboardList == $select.search"   ng-disabled="$ctrl.saveInProgress" /> --> 
```

Comment:
Remove?

Reply:
Yes, all 3 of these sections are removed in the next commit.